### DISCUSSION TITLE
Generated by Tempest EPG Generator https://github.com/K-vanc/Tempest-EPG-Generator

### DISCUSSION THREAD

**Discussion by naijaping** [K-vanc/Tempest-EPG-Generator#179] [General]
Hi Bro,

I just noticed that    <desc lang="en">
Generated by Tempest EPG Generator
https://github.com/K-vanc/Tempest-EPG-Generator</desc> is  being added to all my epg descriptions. I am using Kodi and epg generated via tempest docker image. is this a new behaviour or something is wrong with my docker image.

Thanks

**Comment by K-vanc** [comment]
Hi,

If you are using Tempest and getting an "Generated by Tempest" message, nothing should be wrong in my opinion😄 It shouldn't be in all your descriptions unless you are a very lucky person because it is working randomly with a very low chance of activation. Also it can be in anywhere(title, category, actor etc. etc.) and message can be anything as well..

Anyway, nothing wrong with your setup. I just sent a worldwide "HI" message for the "MASTERS of EPG" who believe that they are very smart by deleting/editing generator-info string from xml and share it as created by their "non-existing" softwares. 

This is not new thing and it exists since the first release of Tempest but I just wanted to enable it first time on last week and disabled a few days ago since message has been taken clearly 😄 so you shouldn't see them on your new runs
